tests/others/test_hub_utils.py

```diff
@@ -15,37 +15,14 @@
 import unittest
 from pathlib import Path
 from tempfile import TemporaryDirectory
-from unittest.mock import Mock, patch
 
-import diffusers.utils.hub_utils
+from diffusers.utils.hub_utils import load_or_create_model_card
 
 
 class CreateModelCardTest(unittest.TestCase):
-    @patch("diffusers.utils.hub_utils.get_full_repo_name")
-    def test_create_model_card(self, repo_name_mock: Mock) -> None:
```

Comment:
Follow the discussion here: https://github.com/huggingface/diffusers/pull/6678#discussion_r1463056454.